phpunit.xml

```diff
@@ -1,24 +1,21 @@
-<phpunit
-    bootstrap="tests/bootstrap.php"
-    colors="true"
-    verbose="true"
->
-    <testsuite name="all">
-        <directory>packages/**/tests</directory>
-        <directory>packages/**/packages/**/tests</directory>
-    </testsuite>
-    <groups>
-        <exclude>
-            <group>api</group>
-        </exclude>
-    </groups>
-    <filter>
-        <whitelist processUncoveredFilesFromWhitelist="true">
-            <directory suffix=".php">packages/*/src</directory>
-        </whitelist>
-    </filter>
-    <!-- to differentiate cache directory of EasyCodingStandards in tests -->
-    <php>
-        <env name="TEST_SUFFIX" value="_tests"/>
-    </php>
+<?xml version="1.0"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="tests/bootstrap.php" colors="true" verbose="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
```

Comment:
The 4-spacing should be kept.

Also indent of options:

```bash
<phpunit
    bootstrap="tests/bootstrap.php"
    colors="true"
    verbose="true"
>
```

Reply:
implemented.